tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -87,13 +88,40 @@ def test_multiple_efs(
     # Names of files that will be written from separate instance. The test checks the cluster nodes can access them.
     existing_efs_filenames = []
     existing_efs_mount_dirs = []
+    iam_authorizations = [False, False, True]
+    encryption_in_transits = [False, True, True]
```

Comment:
This should be two lists of Falses if scheduler is AWS Batch

Reply:
Thanks, I addressed this here https://github.com/Nbot07A/aws-parallelcluster/commit/c1c2f9701ef20f7c217af31c9e19408704b900aa